docker/Makefile

```diff
@@ -0,0 +1,55 @@
+IMAGE=$(USER)/cpu:latest
+all: cat date cpud cpu key.pub Dockerfile
+	docker build . -t $(IMAGE)
+
+key.pub:
+	ssh-keygen -f key -t rsa  -P ""
+
+cpud:
+	CGO_ENABLED=0 go build -o cpud ../cmds/cpud/.
+
+cpu:
+	CGO_ENABLED=0 GOBIN=`pwd` go install  github.com/u-root/cpu/cmds/cpu
+
+date:
+	CGO_ENABLED=0 GOBIN=`pwd` go install  github.com/u-root/u-root/cmds/core/date
+
+cat:
+	CGO_ENABLED=0 GOBIN=`pwd` go install  github.com/u-root/u-root/cmds/core/cat
+
+rundocker:
+	-docker rm cpud_test
+	docker run --mount type=bind,source=$(PWD)/key.pub,target=/key.pub --mount type=bind,source=$(PWD)/key,target=/key --name cpud_test --privileged=true -t -i -p 23:23 $(IMAGE)
+
+run:
```

Comment:
maybe make "run" = "test"?  

i'd have expected "make run" to be the content of "make rundocker"